README.md

```diff
@@ -17,84 +18,91 @@
 ```zsh
 % git clone https://github.com/econia-labs/optivanity.git
 % cd optivanity
-% cargo build --release
+% cargo run --release -- --help
+
+Optivanity: hyper-parallelized vanity address generator for the Aptos blockchain, brought to you by Econia Labs
+
+Usage: optivanity [OPTIONS] <PREFIX>
+
+Arguments:
+  <PREFIX>  Address prefix to match (no leading 0x). Each additional character slows search by 16x
```

Comment:
I'd add backticks to the `0x`. I misread this and thought it was saying "no leading zeroes" for some reason.